src/Appwrite/Utopia/Response/Model/Project.php

```diff
@@ -348,7 +348,7 @@ public function filter(Document $document): Document
             ]);
         }
 
-        $document->setAttribute("providers", $projectProviders);
+        $document->setAttribute("authProviders", $projectProviders);
```

Comment:
I know it wasn't your change, but let's fix style issues like these when we see them (if relatively small in scope)

```suggestion
        $document->setAttribute('authProviders', $projectProviders);
```

Reply:
done this